packages/expo-error-recovery/src/ExpoErrorRecovery.web.ts

```diff
@@ -0,0 +1,19 @@
+const LOCAL_STORAGE_KEY = 'EXPO_ERROR_RECOVERY_STORAGE';
```

Comment:
@EvanBacon what is the convention for internal data that we want to temporarily store?

Reply:
I think this is the only other place we do it https://github.com/expo/expo/blob/99157e58c4551a8d711043ac27c52594c0e53779/packages/expo-constants/src/ExponentConstants.web.ts#L8 so it looks fine